### PR TITLE
adding postlist

### PR DESCRIPTION
**Pull request by vinasxz**
[DEMO LINK](https://vinasxz.github.io/react_static-list-of-posts-js/)

### CODE REVIEW

File: src/components/PostInfo/PostInfo.jsx [hunk 1]
```diff
@@ -1 +1,34 @@
...
+
+      <p className="PostInfo__body">{body}</p>
+
+      <hr />
+
+      {comments.length ? (
+        <CommentList comments={comments} />
+      ) : (
+        <>
+          <hr />
```
Comment (EdPirro) [inline src/components/PostInfo/PostInfo.jsx:28]: Two `hr`s are rendered when there are no comments, only one is needed:

image

File: src/components/PostInfo/PostInfo.jsx [hunk 2]
```diff
@@ -1 +1,34 @@
...
+        <p>
+          {' Posted by  '}
+
+          {user && <UserInfo user={user} />}
+        </p>
+      </div>
+
+      <p className="PostInfo__body">{body}</p>
+
+      <hr />
```
Comment (EdPirro) [inline src/components/PostInfo/PostInfo.jsx:22]: This `hr` is rendered even when there are comments, which is not needed!

**Review by EdPirro — CHANGES_REQUESTED** [review]
Great job, @vinasxz! 👏
There is just one very small issue with your `<hr />`s that I'll ask you to fix but everything else looks good to go!

File: src/components/PostInfo/PostInfo.jsx (hunk 2 above)
Comment (vinasxz) [reply src/components/PostInfo/PostInfo.jsx:22]: I dunno if i get it, but i just delete the line 22.

**Review by jv-aquino — APPROVED** [review]
great job Vinicius! clean and following good practices of react! nice!

**Pull request closed without merging**